Sources/PrimerSDK/Classes/Data Models/ClientToken.swift

```diff
@@ -60,15 +60,21 @@ struct DecodedClientToken: Codable {
     
     func validate() throws {
         if accessToken == nil {
-            throw PrimerError.clientTokenNull
+            let err = PrimerError.invalidClientToken(userInfo: ["file": #file, "class": "\(Self.self)", "function": #function, "line": "\(#line)"])
```

Comment:
wouldn't a developer want to know whether the client token expired as opposed to it being missing for some reason?